### DISCUSSION TITLE
Allo Boss2 dac - no volume control after update rAudio 20240226

### DISCUSSION THREAD

**Discussion by TinkeringB** [rern/rAudio#1612] [Bug Reports]
Re-installed my allo boss 2. 
now running on:
rAudio 20240226
6.6.18-1-rpi aarch64
Raspberry Pi 4B Rev 1.4
BCM2711 • 1.8 GiB
4 x Cortex-A72 @ 1.50 GHz

reproduction steps:
install rAudio-64bit-20240108.img.xz onto sd card and boot.
pacman -Suy
reboot
configure i2s dac : allo boss2
reboot
-> Volume controll not working
Select mixer device "Master" instead of  "Digital"
-> Volume control OK :-)

update rAudio to 20240226
volume in rAudio not working anymore

Now, Mixer is set to "digital" again, and when trying to select "Master",  within seconds, it returns to "Digital".

**Comment by rern** [comment]
Please try:
Settings > Addons > long-press `Update` > `OK` for UPDATE branch

**Reply by TinkeringB** [reply]
Issue resolved :-) 
Hero!
Thank you so much
